### DISCUSSION TITLE
password generator

### DISCUSSION THREAD

**Discussion by AnujPawaadia** [pikanuj12/devops_proj2#2] [General]
hii

**Comment by Dhruv1069** [comment]
hi

**Comment by Dhruv1069** [comment]
Ok so the topic we got for the pair programming activity is password generator

**Comment by AnujPawaadia** [comment]
okkk

**Comment by Dhruv1069** [comment]
we have to make it in python

**Comment by AnujPawaadia** [comment]
any ideas??

**Comment by Dhruv1069** [comment]
i think we should make it gui form

**Comment by AnujPawaadia** [comment]
i have a skeleton ready in command line. we can implement it using tkmodule

**Comment by Dhruv1069** [comment]
ok sounds good

**Comment by AnujPawaadia** [comment]
we can ask the user about thr length of password. that will be helpful for user.

**Comment by Dhruv1069** [comment]
the user can select the tupe of values they need. like uppercase lowercase and all. i think that will be a good thing to add

**Comment by Dhruv1069** [comment]
any other functionalities that we should add?

**Comment by AnujPawaadia** [comment]
how about regenerate and copy option? like the user can regenerate the password and copy them too....

**Comment by Dhruv1069** [comment]
ok we can do that.

**Comment by AnujPawaadia** [comment]
yeah great

**Comment by Dhruv1069** [comment]
also we can add a save option. like the user can save and like modify if they want to or something

**Comment by AnujPawaadia** [comment]
yeah okk. what about used case diagram??

**Comment by Dhruv1069** [comment]
lets finilize the functionalities with the project then we weill make it

**Comment by AnujPawaadia** [comment]
okkk so...
we have this as a rough sketch for project...

1.we r using python and tk module as our GUI 
2.regenerate option
3.copy the password
4.save the desired password
5.the type of values user need. He/She can select according to their need

**Comment by Dhruv1069** [comment]
sounds great. if we get some time later we can add other features as well.

**Comment by AnujPawaadia** [comment]
ye we can do that. so till now lets finalize these and create a used case diagram

I have created one. lets check that

**Comment by Dhruv1069** [comment]
ok great

**Comment by AnujPawaadia** [comment]
![WhatsApp Image 2023-09-25 at 20 02 58](https://github.com/pikanuj12/devops_proj2/assets/124757784/f09c6b20-9669-45e0-9d7b-edebae1420f4)

**Comment by AnujPawaadia** [comment]
is it okay?

**Comment by Dhruv1069** [comment]
yeah amazing

**Comment by AnujPawaadia** [comment]
i have uploaded the "new window" code . please update the widgets ..

**Comment by Dhruv1069** [comment]
i have updated the widgets

**Comment by Dhruv1069** [comment]
completed other changes as well

**Comment by Dhruv1069** [comment]
do chcek them out

**Comment by AnujPawaadia** [comment]
yeah have seen them.
Also i have uploaded my part of code

**Comment by AnujPawaadia** [comment]
so as of now we are done with the code 
i think we should conduct a feedback tomorow.
so that we can analyse the feedback by saturday and implement tthe changes by sunday

**Comment by AnujPawaadia** [comment]
hey i have sahred the feedback link to various groups. 
it will be more helpful if you too share the link.

**Comment by AnujPawaadia** [comment]
https://33v7nw3mky4.typeform.com/to/oDaX5d3H

**Comment by AnujPawaadia** [comment]
https://docs.google.com/spreadsheets/d/1dDezLD_xNzUvoiyt1hkFJBca1JXY0gMZEQDtSpjDJEA/edit?usp=sharing

its the line of feedback and I have created a chart and partially analyzed the feedbacks
so. according to me we need to do some improvement in the graphics part

**Comment by Dhruv1069** [comment]
yeah sure let me look into it and see what we can do

**Reply by AnujPawaadia** [reply]
lets change the background color to some soft solid colors....
but tkinter too have some limitations of background color and others

**Comment by Dhruv1069** [comment]
will try to finish it by sunday night

**Comment by AnujPawaadia** [comment]
![Screenshot (17)](https://github.com/pikanuj12/devops_proj2/assets/124757784/acf993dd-7120-4738-b902-29bd5a0eab4c)

i m going with color

**Comment by AnujPawaadia** [comment]
okk so I have commited the final changes adn i hope u have done your too, if not please do them
